isp/bind/zones/db.mimosa.pi

```diff
@@ -0,0 +1,17 @@
+;
+; BIND data file for mimosa.pi public zone
+;
+$TTL	604800
+mimosa.pi.	IN	SOA	ns1.mimosa.pi. hostmaster.mimosa.pi. (
+		     2018060201		; Serial
+			 604800		; Refresh
+			  86400		; Retry
+			2419200		; Expire
+			 604800 )	; Negative Cache TTL
+;
+mimosa.pi.	IN	NS	ns1.mimosa.pi.
+ns1		IN	A	172.23.44.130
```

Comment:
public DNS zone includes external IP addresses for authoritative server (172.23.44.130) and mail (172.23.44.131)